extensions/3rdparty/tabber/tabber.php

```diff
@@ -44,9 +44,9 @@ function renderTabber( $paramstring, $params, $parser ){
 
 	$htmlTabs = "";
 
-	$arr = explode("|-|", $paramstring);
-	foreach($arr as $tab){
-		$htmlTabs .= buildTab($tab, $parser); # macbre: pass Parser object (refs RT #34513)
+	$arr = explode( "|-|", $paramstring );
+	foreach ( $arr as $tab ) {
+		$htmlTabs .= buildTab( $tab, $parser ); # macbre: pass Parser object (refs RT #34513)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)